Shared/Coordinator/MainCoordinator+Extensions.swift

```diff
@@ -20,7 +20,7 @@ extension MainCoordinator: LaunchManagerDelegate {
     }
 
     func subscribeToUserUpdates() {
-        UserStore.shared.$userDeleted
+        PeopleStore.shared.$userDeleted
```

Comment:
Renaming this to PeopleStore to highlight the generic nature of the data. It can contain both contacts or users, which are both "PersonType" objects.